docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgpartman.md

```diff
@@ -327,3 +327,70 @@ mo=# SELECT create_parent( p_parent_table => 'public.orders',
 ```output
 ERROR:  Partition table public.orders is a colocated table hence registering it to pg_partman maintenance is not supported
 ```
+
+### xCluster
+
+Depending on the type of xCluster deployment, pgPartman may be used.
+
+Note that if you are [using pg_cron to manage tasks](#maintain-partitions-using-pg-cron), you must install it on a separate database that is not part of the xCluster configuration.
```

Comment:
From 2025.1.1:
Only applicable in non-automatic mode. In automatic mode you can have pg_cron on the same db.